pkg/skaffold/deploy/kustomize_test.go

```diff
@@ -34,12 +34,13 @@ import (
 
 func TestKustomizeDeploy(t *testing.T) {
 	tests := []struct {
-		description string
-		cfg         *latest.KustomizeDeploy
-		builds      []build.Artifact
-		commands    util.Command
-		shouldErr   bool
-		forceDeploy bool
+		description           string
+		cfg                   *latest.KustomizeDeploy
+		builds                []build.Artifact
+		commands              util.Command
+		shouldErr             bool
+		forceDeploy           bool
+		useMockKustomizeCheck bool
```

Comment:
From a reading perspective, it might be more intuitive for this to be `kustomizeCmdAbsent`?  Or move the `mockKustomizeCheck` above with a comment as to its use.